### DISCUSSION TITLE
What is the procedure to earn the new achievements badges?

### DISCUSSION THREAD

**Discussion by odin0809** [community/community#20920] [Other Feature Feedback, Questions, & Ideas] [answered]
What to do for earn new badges, which missions I need to complete?

**Comment by Chandan-Choudhury** [comment, marked as answer]
Please follow this [link](https://github.com/drknzz/GitHub-Achievements) to get all the information about new achievement badges.

**Reply by odin0809** [reply]
Ok, thanks for sharing. I will follow the link.
